packages/@aws-cdk/aws-pipes-alpha/lib/logs.ts

```diff
@@ -36,6 +40,52 @@ export enum LogLevel {
   TRACE = 'TRACE',
 }
 
+/**
+ * Log format for `S3LogDestination` logging configuration.
+ *
+ * @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-pipes-pipe-s3logdestination.html#cfn-pipes-pipe-s3logdestination-outputformat
+ */
+export enum S3OutputFormat {
+  /**
+  * PLAIN
+  */
+  PLAIN = 'plain',
+  /**
+  * JSON
+  */
+  JSON = 'json',
+  /**
+  * W3C
+  */
+  W3C = 'w3c',
+}
+
+/**
+ * Properties for `S3LogDestination`.
+ */
+export interface S3LogDestinationProps {
+  /**
+   * The name of the S3 bucket to deliver the log records for the pipe.
```

Comment:
```suggestion
   * The S3 bucket to deliver the log records for the pipe.
```

Reply:
Might also be worth mentioning the cross-account feature here as well, it's a bit more explicit in the CloudFormation documentation with the `BucketOwner` property